### ISSUE TITLE
Bug: Text area and Input style inconsistency in "Post a highlight" form

### ISSUE THREAD

**Issue by FatumaA** [open-sauced/app#2715]
### Describe the bug

The textarea for adding information on the highlight form does not focus when active and the input for addign a highlight is active on form pop up.

As shown in this screenshot:

![Screenshot 2024-02-20 at 12 46 10](https://github.com/open-sauced/app/assets/67555014/19e6fcf9-59de-4e9e-9e42-e8ffb5fa8ffe)

### Steps to reproduce

1. Go to highlights tab
2. Click on the "post a highlight to show your work" input
3. The "post a highlight" form will pop up
4. Notice that the textarea input will not take an active state even when clicked on while the add a highlight link input is active on load

### Browsers

Chrome

### Additional context (Is this in dev or production?)

dev

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by FatumaA** [comment]
.take

**Comment by FatumaA** [comment]
Missed the triage label 🤭

**Comment by takanome-dev** [comment]
Hey @FatumaA, I noticed that on the first launch of the post highlight modal, it appears to be attempting to validate the URL 🤔 

I'll remove the triage label and assign this issue to you. If you're unable to take it on, feel free to unassign yourself!

**Comment by FatumaA** [comment]
Will look into it, thanks!

**Comment by FatumaA** [comment]
I have done a bit of research into accessibility and it appears that it is correct that the most important element should be focussed on load.

However, I tested out the tab focus and the main textarea was not focussable even though all buttons and inputs in the form focussed as I went tabbing along. 

Now redirecting to getting the textarea to focus properly.

Will edit the issue to reflect this.

**Comment by FatumaA** [comment]
> Hey @FatumaA, I noticed that on the first launch of the post highlight modal, it appears to be attempting to validate the URL 🤔
> 
> I'll remove the triage label and assign this issue to you. If you're unable to take it on, feel free to unassign yourself!

Hi,
I removed active/focus state from both unless clicked on or the add-a-highlight input is in error
